Source/VisualStudioTools/Private/VSTestAdapterCommandlet.cpp

```diff
@@ -15,32 +15,34 @@ static constexpr auto RunTestsParam = TEXT("runtests");
 static constexpr auto TestResultsFileParam = TEXT("testresultfile");
 static constexpr auto HelpParam = TEXT("help");
 
-static void GetAllTests(TArray< FAutomationTestInfo >& OutTestList)
+static void GetAllTests(TArray<FAutomationTestInfo> &OutTestList)
 {
-	FAutomationTestFramework& Framework = FAutomationTestFramework::GetInstance();
+	FAutomationTestFramework &Framework = FAutomationTestFramework::GetInstance();
 	Framework.GetValidTestNames(OutTestList);
 }
 
-static void ReadTestsFromFile(const FString& InFile, TArray< FAutomationTestInfo >& OutTestList)
+static void ReadTestsFromFile(const FString &InFile, TArray<FAutomationTestInfo> &OutTestList)
```

Comment:
hmm, since the PR is about formatting, I'll nit this one: the & should be closed to the type and not the identifier: `const FString& InFile`

Same for all the other cases. Check if your settings in VS is not with this misconfigured.

And at some point I want to change the .editorconfig to not have these mismatches anymore.

Reply:
Got it